src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -271,7 +270,7 @@ public static class Extender {
             public static final SparkMaxLimitSwitch.Type SWITCH_TYPE = SparkMaxLimitSwitch.Type.kNormallyClosed;
             public static final double MAX_LENGTH_IN_ROBOT = 0.37;
             public static double MAX_ENTRANCE_LENGTH = 0.054;
-            public static final PIDObject PID = new PIDObject().withKp(60).withKd(2).withMaxPower(1);
+            public static final PIDObject PID = new PIDObject().withKp(0.6).withKd(0.5).withMaxPower(1);
```

Comment:
you might want to clarify that this is a velocity pid and save the old one as a position  pid that is not used for now
same for the elbow